README.md

```diff
@@ -38,7 +38,7 @@ To get a local copy up and running follow these simple example steps.
   - run `rspec` to run the tests.
 
 ### Deployment
-View Project: (https://vast-harbor-92034.herokuapp.com/)
+View Project: 
 
```

Comment:
- [ ] Based on Project requirements, the page should be deployed and have a live version, please bring the link back  and make sure the page works as expected :) 
<img width="582" alt="Screenshot 2022-10-28 at 07 59 27" src="https://user-images.githubusercontent.com/77038610/198506381-cf7eb223-e390-4886-a6b4-27e89af5cab2.png">